main.py

```
@@ -0,0 +1,126 @@
+from fastapi import FastAPI
+from pydantic import BaseModel
+from typing import List
+from utils.model import generate_simulation_result, detect_profanity
```

Ruff (0.11.9)

`utils.model.generate_simulation_result` imported but unused

Remove unused import: `utils.model.generate_simulation_result`

(F401)